apps/web/app/storage/[environmentId]/[accessType]/[fileName]/route.ts

```diff
@@ -81,5 +45,41 @@ export async function GET(
     return responses.unauthorizedResponse();
   }
 
-  return await getFile();
+  return await getFile(environmentId, accessType, fileName);
 }
+
+const getFile = async (environmentId: string, accessType: string, fileName: string) => {
```

Comment:
Can we also move this into the `lib` folder in the same route?
I mostly like having only one (the main) function per component for better readability